### PR TITLE
[#470] Affiliation activation landing page

### PR DESCRIPTION
**Pull request by jswk**
A landing page is shown to the user when she activates an affiliation.
Remove a duplicated Affiliation::Confirm test.

Fixes: #470

### CODE REVIEW

File: app/services/affiliation/confirm.rb [hunk 1]
```diff
@@ -10,16 +10,15 @@ def initialize(user, token)
 
   def call
     @error = nil
-    affiliation = Affiliation.find_by_token(@token)
+    @affiliation = Affiliation.find_by_token(@token)
```
Comment (mkasztelnik) [inline app/services/affiliation/confirm.rb]: I really don't like this change. It changes stateless service into stateful service, what is more, internal implementation is exposed to outside word.

File: app/controllers/affiliation_confirmations_controller.rb [hunk 2]
```diff
@@ -4,14 +4,6 @@ class AffiliationConfirmationsController < ApplicationController
...
-    confirmator = Affiliation::Confirm.new(current_user, params[:at])
-
-    if confirmator.call
-      redirect_to profile_path,
-                  notice: "Affiliation confirmed sucessfully"
-    else
-      redirect_to root_path,
-                  alert: confirmator.error || "Unable to confirm your afiliation"
-    end
+    @confirmator = Affiliation::Confirm.new(current_user, params[:at]).call
```
Comment (mkasztelnik) [inline app/controllers/affiliation_confirmations_controller.rb]: Can't we only change `root_path` into view which shows problems with affiliation activation?

File: app/services/affiliation/confirm.rb (hunk 1 above)
Comment (mkasztelnik) [reply app/services/affiliation/confirm.rb]: If you need more information about failure than please return error status code from the service (e.g. `:ok`, `:not_found`, ...) and in the view render appropriate message (using e.g. I18n support).

Comment (jswk) [reply app/services/affiliation/confirm.rb]: OK, it makes sense. I'll change it to a stateless version

**Pull request merged**